sycl/source/detail/queue_impl.cpp

```diff
@@ -353,17 +353,7 @@ pi_native_handle queue_impl::getNative() const {
 
 bool queue_impl::kernelUsesAssert(const std::string &KernelName,
                                   OSModuleHandle Handle) const {
-  RTDeviceBinaryImage &BinImg = ProgramManager::getInstance().getDeviceImage(
-      Handle, KernelName, get_context(), get_device());
-
-  const pi::DeviceBinaryImage::PropertyRange &AssertUsedRange =
-      BinImg.getAssertUsed();
-  if (AssertUsedRange.isAvailable())
-    for (const auto &Prop : AssertUsedRange)
-      if (Prop->Name == KernelName)
-        return true;
-
-  return false;
+  return ProgramManager::getInstance().kernelUsesAssert(Handle, KernelName);
```

Comment:
Can we remove `kernelUsesAssert` and replace all uses of `kernelUsesAssert` with `ProgramManager::getInstance().kernelUsesAssert`?